### ISSUE TITLE
Lock item position

### ISSUE THREAD

**Issue by z0rgoyok** [hanshengchiu/reorderables#123]
Plz, can I lock some item in this manner?
 
https://youtu.be/tsqT4MMlSsQ

it does not change position. the other elements flow around it

**Comment by yoniker** [comment]
Possible duplication of https://github.com/hanshengchiu/reorderables/issues/89

**Comment by z0rgoyok** [comment]
No, in my case I want to be able to drag the element at will, but so that other elements do not affect it
